### ISSUE TITLE
ColumnHider Button Not Labeled

### ISSUE THREAD

**Issue by arias0906** [SitePen/dgrid#807]
ColumnHider button is not labeled for screen readers. JAWS reads button as unlabeled. For accessibility, the i18nColumnHider.popupLabel should be applied to the button itself as well as an aria-label.
